src/main/java/com/example/bookstore/controller/OrderController.java

```diff
@@ -0,0 +1,92 @@
+package com.example.bookstore.controller;
+
+import com.example.bookstore.dto.order.CreateOrderItemDto;
+import com.example.bookstore.dto.order.OrderItemResponseDto;
+import com.example.bookstore.dto.order.OrderResponseDto;
+import com.example.bookstore.dto.order.UpdateOrderItemDto;
+import com.example.bookstore.model.User;
+import com.example.bookstore.service.OrderItemService;
+import com.example.bookstore.service.OrderService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.Valid;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.HttpStatus;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.security.core.Authentication;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@Tag(name = "Order management", description = "Endpoints for managing orders")
+@RestController
+@RequestMapping(value = "/api/orders")
```

Comment:
Not critical, but you can add this property `server.servlet.context-path=/api` to application.properties and write all RequestMapping's without prefix `/api`